README.md

```diff
@@ -12,107 +12,33 @@
     │   ├── device
     │   ├── x_heep_applications (hw/vendor/esl_epfl_x_heep/sw/applications)       
     │   ├── x_heep_device (hw/vendor/esl_epfl_x_heep/sw/device)
-    │   ├── x_heep_linker (hw/vendor/esl_epfl_x_heep/sw/linker)               
+    │   └── x_heep_linker (hw/vendor/esl_epfl_x_heep/sw/linker)               
     ├── tb
     ├── util
     └── README.md
     
-============================
 
-Minimal configuration of a cgra-x-heep
 
+Minimal configuration of a cgra-x-heep
 ============================
 
 # Prerequisite
 
-The simulation enrironment is based on the x-heep one (https://github.com/esl-epfl/x-heep)
+The simulation enrironment is based on the x-heep one.
 
-1. Install [Conda](https://phoenixnap.com/kb/how-to-install-anaconda-ubuntu-18-04-or-20-04) as described in the link, 
-and create the Conda enviroment with python 3.8:
+Follow the prequisite steps on the [x-heep repository](https://github.com/esl-epfl/x-heep#prerequisite).
```

Comment:
Give the link to the README file inside the vendor folder so we are sure we keep track of the exact steps used to setup the current environment.